src/Raygun.Blazor/Models/BrowserSpecs.cs

```diff
@@ -10,7 +10,7 @@ namespace Raygun.Blazor.Models
     /// This exists because we need to parse some stuff from the browser to construct a proper <see cref="EnvironmentDetails" />
     /// instance. Also the <see cref="EnvironmentDetails" /> is a hot mess of JavaScript serialization inconsistency.
     /// </remarks>
-    internal record BrowserSpecs
+    public record BrowserSpecs
```

Comment:
Made public so it is accesible from the `Raygun.Tests.Blazor.Server` project

Reply:
Fixed it, see [`45d0423` (#53)](https://github.com/MindscapeHQ/raygun4blazor/pull/53/commits/45d0423fae6a4a6eb6866fec59e589b39bab830d)